### ISSUE TITLE
Understanding saving of logits in main.py

### ISSUE THREAD

**Issue by aseembits93** [EnSec-AI/VLM-Uncertainty-Bench#1]
Hi! thank you for sharing your work, it is helping me immensely in my research. I had a quick question - In main.py, you use the get_logits() function which appends the index of the maximum value in the output. This would make the size of the logits as 7. But in 'uncertainty_quantification_via_cp.py', the logits being loaded have a size of 6. Let me know your thoughts on this discrepancy, Thank you!
